.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+### Before requesting gatekeeper:
+
+* [ ] Two review approvals and any prior change requests have been resolved.
+* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it
+      should be force landed and forced-landing tag is set
+
```

Comment:
Suggestions to add:

- the PR owner should ensure the commit message follows the standard before requesting gatekeeper
- the PR owner should also ensure any tests skipped by the ticket being addressed by the PR have been run and passed in the PR